api/openapi.yaml

```
@@ -140,11 +140,11 @@ paths:
           description: Infra ID
           required: true
         - in: query
-          name: include_geom
-          schema:
-            type: boolean
-            default: false
-          description: Whether the railjson should contain geometry information
+          name : exclude_extensions
+          schema :
+            type : boolean
+            default : false
+            description : Whether the railjson should contain extensions
```

You use a yaml formatter. You can use the [redhat.vscode-yaml](https://marketplace.visualstudio.com/items?itemName=redhat.vscode-yaml) vs code extension.

```suggestion
          name: exclude_extensions
          schema:
            type: boolean
            default: false
            description: Whether the railjson should contain extensions
```